jetty-websocket/websocket-core-common/src/main/java/org/eclipse/jetty/websocket/core/internal/TransformingFlusher.java

```diff
@@ -103,8 +126,14 @@ private class Flusher extends IteratingCallback implements Callback
         private FrameEntry current;
 
         @Override
-        protected Action process()
+        protected Action process() throws Throwable
         {
+            try (AutoLock l = lock.lock())
+            {
+                if (failure != null)
+                    throw failure;
```

Comment:
Need a TODO here that his is a temporary fix.     Essentially we need to check a closed boolean here and create an exception if we are processing whilst closed.